### PR TITLE
Add DigitalOcean VPC resource

### PR DESCRIPTION
**Pull request by nemca**
### Description
Add DigitalOcean VPC resource.

### CODE REVIEW

**Comment by nemca** [comment]
I found [issue](https://github.com/tailwarden/komiser/issues/253) for this changes.

File: providers/digitalocean/digitalocean.go [hunk 1]
```diff
@@ -6,12 +6,14 @@ import (
...
 	"github.com/tailwarden/komiser/providers"
 	"github.com/tailwarden/komiser/providers/digitalocean/droplets"
+	"github.com/tailwarden/komiser/providers/digitalocean/network"
 	"github.com/uptrace/bun"
 )
 
 func listOfSupportedServices() []providers.FetchDataFunction {
 	return []providers.FetchDataFunction{
 		droplets.Droplets,
+		network.Vpcs,
```
Comment (mlabouardy) [inline providers/digitalocean/digitalocean.go:16]: related to https://github.com/tailwarden/komiser/pull/291#discussion_r1083928847

File: providers/digitalocean/network/vpcs.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	"github.com/tailwarden/komiser/providers"
+)
+
+func Vpcs(ctx context.Context, client providers.ProviderClient) ([]models.Resource, error) {
+	resources := make([]models.Resource, 0)
+	regionsMap := make(map[string]string)
+
+	regions, _, err := client.DigitalOceanClient.Regions.List(ctx, &godo.ListOptions{})
+	if err != nil {
+		log.Printf("[%s][DigitalOcean] %s", client.Name, err)
```
Comment (mlabouardy) [inline providers/digitalocean/network/vpcs.go:19]: why not return the `err`?

**Review by mlabouardy — CHANGES_REQUESTED** [review]

File: providers/digitalocean/network/vpcs.go (hunk 2 above)
Comment (nemca) [reply providers/digitalocean/network/vpcs.go:19]: I think this is not a critical error and you should not interrupt the function if it was not possible to get the list of regions.
The code below will use the slug name if the fully qualified region name is not found.

Comment (mlabouardy) [reply providers/digitalocean/network/vpcs.go:19]: In this case, can you change the log type, maybe with log.Warn("")

Comment (nemca) [reply providers/digitalocean/network/vpcs.go]: Fixed.

**Review by mlabouardy — APPROVED** [review]
LGTM

**Pull request merged**